packages/toolkit/src/query/core/buildInitiate.ts

```diff
@@ -219,93 +224,117 @@ Features like automatic cache collection, automatic refetching etc. will not be
     endpointName: string,
     endpointDefinition: QueryDefinition<any, any, any, any>
   ) {
-    const queryAction: StartQueryActionCreator<any> = (
-      arg,
-      { subscribe = true, forceRefetch, subscriptionOptions } = {}
-    ) => (dispatch, getState) => {
-      const queryCacheKey = serializeQueryArgs({
-        queryArgs: arg,
-        endpointDefinition,
-        endpointName,
-      })
-      const thunk = queryThunk({
-        subscribe,
-        forceRefetch,
-        subscriptionOptions,
-        endpointName,
-        originalArgs: arg,
-        queryCacheKey,
-      })
-      const thunkResult = dispatch(thunk)
-      middlewareWarning(getState)
-      const { requestId, abort } = thunkResult
-      const statePromise = Object.assign(
-        thunkResult.then(() =>
-          (api.endpoints[endpointName] as ApiEndpointQuery<any, any>).select(
-            arg
-          )(getState())
-        ),
-        {
-          arg,
-          requestId,
+    const queryAction: StartQueryActionCreator<any> =
```

Comment:
yay prettier... :facepalm: 
please activate that checkbox there :smile: 
![image](https://user-images.githubusercontent.com/4282439/125124851-0b89f080-e0f9-11eb-858c-bc2a1361e8d7.png)